poly-commitment/benches/ipa.rs

```diff
@@ -8,6 +8,69 @@ use poly_commitment::{
     commitment::CommitmentCurve, ipa::SRS, utils::DensePolynomialOrEvaluations, PolyComm, SRS as _,
 };
 
+fn benchmark_msm(c: &mut Criterion) {
```

Comment:
Nit: I would add in the name that it is with Vesta.

Reply:
Same comment as https://github.com/o1-labs/proof-systems/pull/2969#discussion_r1965248308